lang/csharp/src/apache/codegen/AvroGen.cs

```diff
@@ -43,9 +43,9 @@ static void Main(string[] args)
                 {
                     if (i + 1 >= args.Length)
                     {
-                        Console.WriteLine("Missing path to protocol file");
+                        Console.Error.WriteLine("Missing path to protocol file");
                         Usage();
-                        return;
+                        return -1;
```

Comment:
A return code of 1 is more common for a general error like this. `avro-tools.jar` returns exits with 1 in these scenarios as well. Let's switch all the `-1` returns to `1`.